File: lecture1/pset1/mario_more/mario.c
#include <cs50.h> 
#include <stdio.h>

int main(void) 
{
    int h;
    
    do 
    {
        h = get_int("Height: ");
        
        if (h < 1 || h > 8) 
        {
            printf("It must be some number between 1 and 8.");
        }
    } while (h < 1 || h > 8);
    
    for (int i = 1; i <= h; i++) //i: layer of the tower
    {
        for (int s = h - i; s > 0; s--) //s: space
        {
            printf(" ");
        }
        
        for (int j = 1; j <= i; j++) //j: brick
        {
            printf("#");
        }
        
        printf("  ");
        
        for (int j2 = 1; j2 <= i; j2++) //j: brick2
        {
            printf("#");
        }
        
        printf("\n");
    }
}

File: lecture6/pset6/mario/less/mario.py
import cs50
while True:
    height = cs50.get_int("Height: ")
    if height >= 1 and height <= 8:
        break
    
for line in range(1, height+1):
    for gap in range(height-line, 0, -1):
        print(' ', end='')
        
    for brick in range(1, line+1):
        print('#', end='')
        
    print()